net/Socket.cpp

```diff
@@ -1303,13 +1306,18 @@ bool StreamSocket::parseHeader(const char *clientName,
 {
     assert(map._headerSize == 0 && map._messageSize == 0);
 
+    constexpr std::chrono::duration<float, std::milli> delayMax =
```

Comment:
Why convert micros into fractional millis? Seems having duration values as microsecond integers is more straightforward than this conversion.